MergeHRISClient/model_utils.py

```diff
@@ -1488,10 +1494,19 @@ def model_to_dict(model_instance, serialize=True):
                 # exist in attribute_map
                 attr = model_instance.attribute_map.get(attr, attr)
             if isinstance(value, list):
-                result[attr] = list(map(
-                    lambda x: model_to_dict(x, serialize=serialize)
-                    if hasattr(x, '_data_store') else x, value
-                ))
+               if not value:
+                   # empty list or None
+                   result[attr] = value
+               else:
+                   res = []
+                   for v in value:
+                       if isinstance(v, PRIMITIVE_TYPES) or v is None:
+                           res.append(v)
+                       elif isinstance(v, ModelSimple):
+                           res.append(v.value)
+                       else:
+                           res.append(model_to_dict(v, serialize=serialize))
+                   result[attr] = res
```

Comment:
^ is this from a new generator version?

Reply:
Yeah i was surprised that this file had changed, i definitely use the same generator version https://github.com/merge-api/merge-backend/blob/develop/docker-compose.yml#L112 of `5.1.1`

Reply:
tbh we should probably git ignore this file unless we have explicit changes we want to make, it causes a lot of problems

Reply:
All set, although I'm going to hold off on the gitignore for now